--- Connext-5.1.0-RHEL6_64.sh.orig	2015-04-10 15:44:46.000000000 -0700
+++ Connext-5.1.0-RHEL6_64.sh	2015-04-10 15:44:37.000000000 -0700
@@ -28,7 +28,7 @@
 DEF_MYSHELL=bash
 DEF_ENVFILE=$HOME/.bashrc
 menu_env() {
-  read ENV
+  #read ENV
   if [ -z $ENV ]; then
    ENV=$DEF_ENV
   fi
@@ -48,7 +48,8 @@
 menu_license() {
   echo ""
   echo "Do you accept this Agreement? [y/n]"
-  read ACCEPT
+  #read ACCEPT
+  ACCEPT=y
   case $ACCEPT in
     y)
       ;;
@@ -71,7 +72,8 @@
   fi
   echo ""
   echo "Do you want to open the Getting Started Guide? [y/n]"
-  read ACCEPT
+  #read ACCEPT
+  ACCEPT=n
   case $ACCEPT in
     [yY])
       $PDFREADER $GSGDOC &
@@ -90,7 +92,8 @@
   LAUNCHER_START=$DEST/$TOP_LEVEL_DIR/RTI_Launcher_$LAUNCHER_V/scripts/rtilauncher
   echo ""
   echo "Do you want to start RTI Connext Launcher now? [y/n]"
-  read ACCEPT
+  #read ACCEPT
+  ACCEPT=n
   case $ACCEPT in
     [yY])
       $LAUNCHER_START &
@@ -109,7 +112,7 @@
   echo "Select wich shell do you want to configure: [$DEF_MYSHELL]"
   echo " 1. bash"
   echo " 2. tcsh"
-  read ANSWER
+  #read ANSWER
   if [ -z $ANSWER ]; then
    ANSWER=1
   fi
@@ -147,15 +150,15 @@
 echo "================================================="
 echo ""
 echo "Press enter to read the license agreement"
-read LICENSE
-tail -n+$LICSTART $THIS | head -n$DIFF | more
+#read LICENSE
+#tail -n+$LICSTART $THIS | head -n$DIFF | more
 
 menu_license
 
 echo ""
 echo "Specify a location for the installation (the directory \"$TOP_LEVEL_DIR\""
 echo "will be created in the location you specify): [$DEF_DEST]"
-read DEST
+#read DEST
 if [ -z $DEST ]; then
    DEST=$DEF_DEST
 fi

